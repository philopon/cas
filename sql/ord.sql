SELECT '24311-42-1'::cas > '24311-42-1'::cas;
SELECT '24311-42-1'::cas >= '24311-42-1'::cas;
SELECT '24311-42-1'::cas < '24311-42-1'::cas;
SELECT '24311-42-1'::cas <= '24311-42-1'::cas;

SELECT '34311-42-1'::cas > '24311-42-1'::cas;
SELECT '34311-42-1'::cas >= '24311-42-1'::cas;
SELECT '34311-42-1'::cas < '24311-42-1'::cas;
SELECT '34311-42-1'::cas <= '24311-42-1'::cas;

SELECT '24311-45-1'::cas > '24311-42-1'::cas;
SELECT '24311-45-1'::cas >= '24311-42-1'::cas;
SELECT '24311-45-1'::cas < '24311-42-1'::cas;
SELECT '24311-45-1'::cas <= '24311-42-1'::cas;

SELECT '24311-42-0'::cas > '24311-42-1'::cas;
SELECT '24311-42-0'::cas >= '24311-42-1'::cas;
SELECT '24311-42-0'::cas < '24311-42-1'::cas;
SELECT '24311-42-0'::cas <= '24311-42-1'::cas;

