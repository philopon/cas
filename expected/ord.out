SELECT '24311-42-1'::cas > '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-42-1'::cas >= '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '24311-42-1'::cas < '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-42-1'::cas <= '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '34311-42-1'::cas > '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '34311-42-1'::cas >= '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '34311-42-1'::cas < '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '34311-42-1'::cas <= '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-45-1'::cas > '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '24311-45-1'::cas >= '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '24311-45-1'::cas < '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-45-1'::cas <= '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-42-0'::cas > '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-42-0'::cas >= '24311-42-1'::cas;
 ?column? 
----------
 f
(1 row)

SELECT '24311-42-0'::cas < '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

SELECT '24311-42-0'::cas <= '24311-42-1'::cas;
 ?column? 
----------
 t
(1 row)

